raiden/tests/scenarios/bf2_long_running.yaml

```diff
@@ -0,0 +1,332 @@
+version: 2
+
+settings:
+  gas_price: "fast"
+  chain: goerli
+  services:
+    pfs:
+      url: https://pfs-goerli.services-dev.raiden.network
+    udc:
+      enable: true
+      token:
+        deposit: true
+
+token:
+  address: "0x62083c80353Df771426D209eF578619EE68D5C7A"
+  balance_fund: 10_000_000_000_000_000_000
+
+nodes:
+  mode: managed
+  count: 5
+  raiden_version: local
+  enable-monitoring: false
+  default_options:
+    gas-price: fast
+    environment-type: development
+    default-settle-timeout: 40
+    default-reveal-timeout: 20
+    proportional-fee:
+      - "0x62083c80353Df771426D209eF578619EE68D5C7A"
+      - 0
+    proportional-imbalance-fee:
+      - "0x62083c80353Df771426D209eF578619EE68D5C7A"
+      - 0
+
+# This is the bf2 long running scenario. This scenario mimics user behaviour for 
+# opening channels, depositing, transferring, waiting keeping the raiden node alive for 
+# a long time in the process checking whether all distributed system in raiden network 
+# work accurately with a sufficiently dirty state of blockchain for a long time.
+
+scenario:
+  serial:
+    name: "Long running scenario"
+    tasks:
+      - parallel:
+          name: "Create channels"
+          tasks:
+            - open_channel: {from: 0, to: 1, total_deposit: 0, expected_http_status: 201}
+            - open_channel: {from: 1, to: 2, total_deposit: 0, expected_http_status: 201}
+            - open_channel: {from: 2, to: 3, total_deposit: 0, expected_http_status: 201}
+      - serial:
+          name: "Allow time for nodes to learn about their channels"
+          tasks:
+            - wait: 30
+      - parallel:
+          name: "Verify if channels were opened correctly"
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 0, balance: 0,  state: "opened"}
+            - assert: {from: 1, to: 0, total_deposit: 0, balance: 0,  state: "opened"}
+            - assert: {from: 1, to: 2, total_deposit: 0, balance: 0,  state: "opened"}
+            - assert: {from: 2, to: 1, total_deposit: 0, balance: 0,  state: "opened"}
+            - assert: {from: 2, to: 3, total_deposit: 0, balance: 0,  state: "opened"}
+            - assert: {from: 3, to: 2, total_deposit: 0, balance: 0,  state: "opened"}
+      - parallel:
+          name: "0 1 2 3 make a deposit of 100 to their channels"
+          tasks:
+            - deposit: {from: 0, to: 1, total_deposit: 100_000_000_000_000_000}
+            - deposit: {from: 1, to: 0, total_deposit: 100_000_000_000_000_000}
+            - deposit: {from: 1, to: 2, total_deposit: 100_000_000_000_000_000}
+            - deposit: {from: 2, to: 1, total_deposit: 100_000_000_000_000_000}
+            - deposit: {from: 2, to: 3, total_deposit: 101_000_000_000_000_000}
```

Comment:
This amount is different from the others. Maybe add a comment noting it and telling why.